### ISSUE TITLE
Revisit checks

### ISSUE THREAD

**Issue by MonikaFu** [RMI-PACTA/r2dii.plot#516]
Right now there seem to be a lot of checks on different OSes which might be an overkill

**Comment by jdhoffa** [comment]
The [checks](https://github.com/RMI-PACTA/r2dii.plot/pull/532/checks) used in `r2dii.plot` () are in line with what we [checks](https://github.com/RMI-PACTA/r2dii.analysis/pull/451/checks) we use for `r2dii.analysis`. Not saying that means we need to keep them, but it at least means their in-line with one another. 

Note: there is a discussion going on in parallel around standardizing our checks across the PACTA tech team.
Discussion is happening in the context of this repo: https://github.com/RMI-PACTA/demo_actions/issues

I would recommend closing this issue in favour of an issue/ discussion in that repo :-)

**Comment by MonikaFu** [comment]
@jdhoffa which issue exactly that is? I'm happy to move it there if it covers what I wanted to discuss here. Basically many times I had checks breaking for no reason and my intuition is that partly it is due to the fact that there are so many. Not sure if it is necessary to have so many checks for each PR. Maybe only for some when we make a release?

**Comment by jdhoffa** [comment]
Well it is not an issue, but the repository itself, where we plan to discuss what workflows to use for all repositories. 

Eventually, we will try to base all of our repositories workflows off of this central place so that we don't have a bunch of different versions of the same thing

**Comment by jdhoffa** [comment]
I just made a new issue covering it here: https://github.com/RMI-PACTA/demo_actions/issues/31

**Comment by jdhoffa** [comment]
We will discuss it as well on February 1st I believe. I think @AlexAxthelm is going to send an invite.

**Comment by MonikaFu** [comment]
Closing this issue as the number and breadth of checks will be decided for all our repos when we solve this issue: https://github.com/RMI-PACTA/demo_actions/issues/31.
